### PR TITLE
[RFR] fixing retry fixture and test_retry_migration_plan

### PR DESCRIPTION
**Pull request by mnadeem92**
There is an issue with below Selenium send key"
self.browser.send_keys(Keys.ENTER, self)", So whenever for
the first time send key gets called, it cause the reload of
browser and which landed in all page of the said collection
and throw the exception "NoSuchElementException".

which causes the failure of TC, This PR will works as work around
to resolve the failure.

Signed-off-by: mnadeem92 <mnadeem@redhat.com>

{{  pytest: cfme/tests/v2v/test_v2v_cancel_migrations.py -k "test_retry_migration_plan"  --use-provider rhv-ims  --use-provider vsphere67-ims --provider-limit 2 -v }}

### CODE REVIEW

**Comment by dajoRH** [comment]
I detected some fixture changes in commit 31c04f5e82c44e5ae2037405c70ec332632702fb

The *local* fixture **`cancel_migration_plan`** is used in the following files:
- **cfme/tests/v2v/test_v2v_cancel_migrations.py**
    - *test_retry_migration_plan*

Please, consider creating a PRT run to make sure your fixture changes do not break existing usage :smiley:

File: cfme/tests/v2v/test_v2v_cancel_migrations.py [hunk 1]
```diff
@@ -43,21 +45,26 @@ def cancel_migration_plan(appliance, provider, mapping_data_vm_obj_mini):
         target_provider=provider,
         vm_list=mapping_data_vm_obj_mini.vm_list
     )
+
     if migration_plan.wait_for_state("Started"):
+
+        # Cancell migration after disk migration elapsed a 4 minute time duration
```
Comment (sshveta) [inline cfme/tests/v2v/test_v2v_cancel_migrations.py]: 'Cancel' -Typo

File: cfme/tests/v2v/test_v2v_cancel_migrations.py [hunk 2]
```diff
@@ -43,21 +45,26 @@ def cancel_migration_plan(appliance, provider, mapping_data_vm_obj_mini):
...
+        migration_plan.in_progress(wait_time=4)
```
Comment (sshveta) [inline cfme/tests/v2v/test_v2v_cancel_migrations.py]: I am doubtful about calling migration_plan.in_progress when you are waiting for plan to start .

         migration_plan.wait_for_state("Started"): 
Just check once if this is working

Comment (mnadeem92) [reply cfme/tests/v2v/test_v2v_cancel_migrations.py]: yes, it should work as wait_for_state("Started") make sure the plan gets approved and moved to in-progress.

File: cfme/tests/v2v/test_v2v_cancel_migrations.py [hunk 3]
```diff
@@ -220,15 +227,22 @@ def test_retry_migration_plan(cancel_migration_plan,
...
+    except NoSuchElementException:
+        view = navigate_to(migration_plan, "Complete")
     # Retry Migration
     view.plans_completed_list.migrate_plan(migration_plan.name)
     assert migration_plan.wait_for_state("Started")
 
     # Automating BZ 1755632
+    retry_interval = 15
+    if appliance.version < '5.11':
+        retry_interval = 60
```
Comment (valaparthvi) [inline cfme/tests/v2v/test_v2v_cancel_migrations.py]:
```suggestion
retry_interval = 60 if appliance.version < "5.11" else 15
```

**Comment by dajoRH** [comment]
I detected some fixture changes in commit f43e48ddbd5dcea4ef1ed627db7857cad2feb412

The *local* fixture **`cancel_migration_plan`** is used in the following files:
- **cfme/tests/v2v/test_v2v_cancel_migrations.py**
    - *test_retry_migration_plan*

Please, consider creating a PRT run to make sure your fixture changes do not break existing usage :smiley:

File: cfme/v2v/migration_plans.py [hunk 4]
```diff
@@ -417,6 +418,12 @@ def _in_progress():
                             format(plan_name=self.name, visibility=is_plan_visible,
                                    message=new_msg)
                         )
+                        if wait_time:
```
Comment (mshriver) [inline cfme/v2v/migration_plans.py]: This is inconsistent with the rest of the framework in a few ways.

1. Use `wait` kwarg, which we discussed and agreed on for use (see create_view, handle_alert)
2. The default for wait is arguable, but I would use `None`
3. The value passed to `wait` should be seconds, not minutes
4. the docblock should explain the arg

Comment (mshriver) [reply cfme/v2v/migration_plans.py]: Re-reading how this is used, my point #4 is even more important.

Its not a wait, and using `wait` or `wait_time` here is very misleading.

What you're trying to capture here is the _elapsed_ time of the in progress plan.

The name of the method in the first place is completely misleading - as this function does not return a boolean for whether the plan is in progress, it waits for the plan to be _not_ in progress.

The kwarg here should be `elapsed_time`, and the method name should be something like `wait_for_progress`, where by default it waits until the plan is no longer in progress (current behavior) or it waits until a specific elapsed time (what you're trying to add).

Comment (sshveta) [reply cfme/v2v/migration_plans.py]: @mshriver @mnadeem92 : we can also consider creating a small separate method to cancel_migration() that will wait for sometime , cancel migration and then check if the Vm's are migrating anymore and return status if we don't want to modify the current in_progress method

**Comment by dajoRH** [comment]
I detected some fixture changes in commit 46e376f29c5a97b8db80dbb0d9dc2b8b2ed3211f

The *local* fixture **`cancel_migration_plan`** is used in the following files:
- **cfme/tests/v2v/test_v2v_cancel_migrations.py**
    - *test_retry_migration_plan*

Please, consider creating a PRT run to make sure your fixture changes do not break existing usage :smiley:

**Pull request merged**